controllers/organizations/event.js

```diff
@@ -0,0 +1,76 @@
+const Event = require('../../models/Event');
+
+module.exports = {
+  getOrgEvents: () => {
+    /* TODO: implement me */
+  },
+  getOrgEvent: () => {
+    /* TODO: implement me */
+  },
+  createOrgEvent: async (req, res) => {
+    const {
+      isPublished,
+      title,
+      location,
```

Comment:
How to create locations for new queries in create or update

Reply:
ask user to either select from stored locations or create a new one (call create location route to get newly created location ID and submit with create event API call)